java-checks/src/main/java/org/sonar/java/checks/LeastSpecificTypeCheck.java

```diff
@@ -84,17 +83,20 @@ private static boolean isStringType(Type type) {
     return type.isUnknown() || type.is("java.lang.String");
   }
 
-  private void handleParameter(Symbol parameter, boolean springInjectionAnnotated) {
-    Type parameterType = parameter.type();
+  private void handleParameter(Symbol parameterSymbol, boolean springInjectionAnnotated) {
+    Type parameterType = parameterSymbol.type();
```

Comment:
Same here: the method got some refactoring but is otherwise untouched. This distracts from the _actual_ change this PR is about.